src/tablib/formats/_xlsx.py

```diff
@@ -51,6 +57,11 @@ def export_set(cls, dataset, freeze_panes=True, invalid_char_subst="-"):
         )
 
         cls.dset_sheet(dataset, ws, freeze_panes=freeze_panes)
+        if isinstance(column_width, str) and column_width != "adaptive":
+            raise ValueError(f"Unsupported value `{column_width}` passed to `column_width` "
+                             f"parameter. It supports 'adaptive' or integer values")
```

Comment:
No need for f-string here:
```suggestion
                             "parameter. It supports 'adaptive' or integer values")
```